platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/state/DefaultStateManagementComponent.java

```diff
@@ -478,7 +478,7 @@ public AutoCloseableWrapper<SignedState> find(final @NonNull Predicate<SignedSta
     }
 
     /**
-     * This observer is called when the most recent signed state is requested to be dumped to disk.
+     * This observer is called a signed state is requested to be dumped to disk.
```

Comment:
" ...called *when* a signed state is requested... "

feel free to merge without fixing though, since at least it is no longer explicitly incorrect

Reply:
Today we overlook grammatical errors, tomorrow there is anarchy and riots in the streets. MUST FIX!